### ISSUE TITLE
Mix Mode - Safari caret position after new tag

### ISSUE THREAD

**Issue by MenahemGershovitz** [yairEO/tagify#772]
Hi Yair
still not working on Safari
did you find some bug about that issue ?
Thank you

**Comment by yairEO** [comment]
just checked in my Safari (`v14.0.3`), MacOS, and I don't see a problem with the caret placement after adding a tag in *mix-mode*

**Comment by yairEO** [comment]
Now I see. You did not explain the steps needed to be done in order to see the bug.

It only happens when selecting by mouse/touch.

Is this bug important for your company?

**Comment by ShimshonMarketing66** [comment]
Hi Yair Thank you for Response 
yes it when I select the by touch and yes its important for me Im using IONIC for building an app all is working great insteatof this .

**Comment by MenahemGershovitz** [comment]
do you have the solution ?

**Comment by yairEO** [comment]
Hi, I did not look for a solution, I am quite busy lately.. if it's important to your company please feel free to hire me try and fix this urgently

**Comment by andriy89382** [comment]
Hi, this doesn't work for me too. Did you find a solution?

**Comment by yairEO** [comment]
No, I don't have access to Safari browser and don't have time for this... sorry... but I will be happy to see a PR if someone is interested

**Comment by kkragoth** [comment]
@andriy89382 - I'm using tagify in React, my current workaround is to set in settings mixMode.insertAfterTag: '' and in onAdd callback do: 'setTimeout(() => document.execCommand('insertText', false, ' ', 100)'

It kinda solves issue that I was facing in adding in safair.

However deleting in mixed content is broken on safari, this is my typing on Version 18.0.1 (20619.1.26.31.7) on examples:
https://youtu.be/8cyZiVpdCGI
